src/Graphics/Image/Processing/Gabor.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE BangPatterns #-} 
+
+module Graphics.Image.Processing.Gabor where
+
+import Control.Monad (forM_, when)
+import Control.Monad.ST
+import Data.STRef 
+
+import Prelude as P hiding (subtract)
+import Graphics.Image.Processing.Filter
+import Graphics.Image.Interface as I
+import Graphics.Image
+import Graphics.Image.Types as IP
+import Graphics.Image.ColorSpace (X)
+import Data.Complex (Complex((:+)))
+
+gaborfn
+  :: (RealFloat p, Fractional p) => p -> p -> p -> p -> p -> p -> p -> Complex p  
+gaborfn λ θ ψ σ γ x y = exp ( (-0.5) * ((x'^2 + γ^2*y'^2) / (σ^2)) :+ 0) * exp ( 0 :+ (2*pi*(x'/λ+ψ)) )
```

Comment:
`(2*pi*(x'/λ+ψ))`, that's incorrect, should be `(2*pi*x'/λ+ψ)`

Reply:
That's great, you copy pasted it from Wikipedia, even managed to copy it with an error.

Reply:
Very sorry sir, will correct that. I was unaware of the exact formula and found the haskell version on wiki. Will correct it.